packages/create-cloudflare/e2e-tests/workers.test.ts

```diff
@@ -109,15 +168,7 @@ describe
 							return {
 								...testConfig,
 								name: `${testConfig.name ?? testConfig.template}-${variant.toLowerCase()}`,
-								promptHandlers: [
-									{
-										matcher: /Which language do you want to use\?/,
-										input: {
-											type: "select",
-											target: variant,
-										},
-									},
-								],
```

Comment:
We are now passing the `--lang` arg directly instead of trying to answer the prompt.